pennylane_lightning/lightning_qubit/_measurements.py

```diff
@@ -72,7 +70,7 @@ class LightningMeasurements:
 
     def __init__(
         self,
-        qubit_state: LightningStateVector,
+        qubit_state,
```

Comment:
Why remove that?

Reply:
`LightningStateVector` is solely imported for this type annotation, but `LightningMeasurements` is required in `_state_vector.py` so removing this avoids a cyclic dependency.

Reply:
Makes sense! Thank you for that!